src/backend/commands/player/kick/multi_kick.cpp

```diff
@@ -18,12 +18,12 @@ namespace big
 		virtual void execute(player_ptr player, const command_arguments& _args, const std::shared_ptr<command_context> ctx) override
 		{
 			if (g_player_service->get_self()->is_host())
-				dynamic_cast<player_command*>(command::get("breakup"_J))->call(player, {}),
-				    dynamic_cast<player_command*>(command::get("hostkick"_J))->call(player, {});
-
-			dynamic_cast<player_command*>(command::get("oomkick"_J))->call(player, {});
-			script::get_current()->yield(700ms);
+				dynamic_cast<player_command*>(command::get("breakup"_J))->call(player, {});
 
+			if (player->is_host()) {
+				dynamic_cast<player_command*>(command::get("oomkick"_J))->call(player, {});
+				script::get_current()->yield(700ms);
```

Comment:
Why are we yielding? If we're oom-kicking, then we won't be following up with a desync because you're already verified that the kick target is the host, so your next check for !player->is_host() will return false and therefore never call the desync.